pkg/deployments/scaler.go

```diff
@@ -116,6 +116,15 @@ func (s *scaler) compareScales(current, desired int32) {
 	}
 }
 
+func (s *scaler) StopScaleDownTimer() {
```

Comment:
I would prefer to have a general `.Stop()` method on scaler if scaler needs some cleanup to be done before deletion.